rsmtool/convert_feature_json.py

```diff
@@ -15,7 +16,7 @@
 import pandas as pd
 
 
-def convert_feature_json_file(json_file, output_file, delete=False):
+def convert_feature_json_file(json_file: str, output_file: str, delete=False) -> None:
```

Comment:
```suggestion
def convert_feature_json_file(json_file: str, output_file: str, delete: bool=False) -> None:
```

Reply:
Yeah, @mulhod asked me the same thing :) I was avoiding adding hints to things that were explicitly and obviously boolean but since both of you suggested this, I think I should just do it. Will fix.

Reply:
I realized at some point that it's not a typo.. it is obvious but it's not the only obvious thing :)
how did mypy not complain?

Reply:
It doesn't complain because it can infer the type successfully without the hint :)